pkg/driver/sqlite/sqlite_test.go

```diff
@@ -233,6 +233,7 @@ func TestSQLiteSelectMigrations(t *testing.T) {
 	// test limit param
 	migrations, err = drv.SelectMigrations(db, 1)
 	require.NoError(t, err)
+
```

Comment:
You may want to remove this space, to avoid creating conflicts where there is no need to :-)

Reply:
fixed!